src/components/CallItem/index.js

```diff
@@ -65,8 +65,16 @@ export default class CallItem extends Component {
       selected: this.getInitialContactIndex(),
       isLogging: false,
       isCreating: false,
+      loading: true,
     };
     this._userSelection = false;
+    setTimeout(() => {
```

Comment:
Set time out in `componentDidMount` and clean timeout in unmounted ? if `componentDidMount` is called after 10, it will alway be loading.